onward/app/controllers/MostPopularController.scala

```diff
@@ -31,7 +31,10 @@ object MostPopularController extends Controller with Logging with ExecutionConte
     val sectionPopular: Future[List[MostPopular]] = if (path.nonEmpty) lookup(edition, path).map(_.toList) else Future(Nil)
 
     sectionPopular.map { sectionPopular =>
-      val mostPopular = globalPopular.toList ++ sectionPopular
+      val sectionFirst = sectionPopular ++ globalPopular
```

Comment:
Do you need to .toList on this line too on the globalPopular? (just guessing here)